troposphere/static/js/components/images/ImageDetailsPage.react.js

```diff
@@ -15,7 +15,7 @@ define(function (require) {
     renderBody: function(){
       var image = stores.ImageStore.get(Number(this.getParams().imageId)),
         tags = stores.TagStore.getAll(),
-        userLoggedIn = context.profile,
+        userLoggedIn = (context.profile && context.profile.get('selected_identity')),
```

Comment:
Occurrences of `context.profile.get('selected_identity')` could be changed to testing for _attribute_ `'username'`. In the `jetstream` branch, I believe I added as "hasLoggedInUser()" - which would prevent these types of statements being littered all over the place. 

I will find that commit - backport via cherry-pick and get it in place.